api/operator.go

```diff
@@ -63,6 +71,20 @@ type KeyringResponse struct {
 	NumNodes int
 }
 
+// AutopilotConfiguration is used for querying/setting the Autopilot configuration
+type AutopilotConfiguration struct {
+	// DeadServerCleanup controls whether to remove dead servers from the Raft
+	// peer list when a new server joins
+	DeadServerCleanup bool
```

Comment:
sounds good, this was more of a placeholder name anyway since I didn't want to dwell on it too long